### PR TITLE
Implement new questions using facebook reactions. Fix #31

### LINKED ISSUE #31: Use a proper logging framework on the server
- https://meteorhacks.com/logging-support-for-meteor
- https://atmospherejs.com/infinitedg/winston
- https://www.loggly.com/blog/managing-a-meteor-application-in-production-three-real-log-management-use-cases/
- http://observatoryjs.com/
- https://atmospherejs.com/superstringsoft/observatory
- https://github.com/tbknl/meteor-trail

### PR DESCRIPTION
**Pull request by vincenzobaz**
Currently it only shows the generic "Who reacted ..." because that's the only type of reactions received by the server.

### CODE REVIEW

File: meteor/imports/client/helpers/getQuestionTitleByType.ts [hunk 1]
```diff
@@ -10,7 +10,14 @@ const en = {
   ORDPostCommentsNumber: 'Order these posts according to the number of comments they got: least comments first',
   ORDPostLikesNumber: 'Order these posts according to the number of likes: fewest likes on top',
   ORDPostTime: 'Order theses posts after the time you posted them: oldest on top',
-  MCWhoLikedYourPost: 'Who liked this post you made?',
+  MCWhoReactedToYourPost: 'Who reacted (Like, Wow, Haha, Love, Sad, Angry, Thankful) to this post you made?',
+  MCWhoReactedToYourPostWithLIKE: 'Who liked this post you made?',
+  MCWhoReactedToYourPostWithWOW: 'Who reacted with Wow to this post you made?',
+  MCWhoReactedToYourPostWithHAHA: 'Who reacted with Haha to this post you made?',
+  MCWhoReactedToYourPostWithLOVE: 'Who reacted with Love to this post you made?',
```
Comment (Rabyss) [inline meteor/imports/client/helpers/getQuestionTitleByType.ts]: Who reacted with Love -> Who loved

File: meteor/imports/client/helpers/getQuestionTitleByType.ts [hunk 2]
```diff
@@ -10,7 +10,14 @@ const en = {
...
+  MCWhoReactedToYourPostWithSAD: 'Who reacted with Sad to this post you made?',
+  MCWhoReactedToYourPostWithANGRY: 'Who reacted with Angry to this post you made?',
+  MCWhoReactedToYourPostWithTHANKFUL: 'Who reacted with Thankful to this post you made?',
```
Comment (Rabyss) [inline meteor/imports/client/helpers/getQuestionTitleByType.ts]: Who was thankful for your post?

File: meteor/imports/client/helpers/getQuestionTitleByType.ts [hunk 3]
```diff
@@ -10,7 +10,14 @@ const en = {
...
   ORDPostLikesNumber: 'Order these posts according to the number of likes: fewest likes on top',
   ORDPostTime: 'Order theses posts after the time you posted them: oldest on top',
-  MCWhoLikedYourPost: 'Who liked this post you made?',
+  MCWhoReactedToYourPost: 'Who reacted (Like, Wow, Haha, Love, Sad, Angry, Thankful) to this post you made?',
+  MCWhoReactedToYourPostWithLIKE: 'Who liked this post you made?',
+  MCWhoReactedToYourPostWithWOW: 'Who reacted with Wow to this post you made?',
+  MCWhoReactedToYourPostWithHAHA: 'Who reacted with Haha to this post you made?',
+  MCWhoReactedToYourPostWithLOVE: 'Who reacted with Love to this post you made?',
+  MCWhoReactedToYourPostWithSAD: 'Who reacted with Sad to this post you made?',
+  MCWhoReactedToYourPostWithANGRY: 'Who reacted with Angry to this post you made?',
```
Comment (Rabyss) [inline meteor/imports/client/helpers/getQuestionTitleByType.ts]: Who was angry at your post?

File: meteor/imports/client/helpers/getQuestionTitleByType.ts [hunk 4]
```diff
@@ -10,7 +10,14 @@ const en = {
   ORDPostCommentsNumber: 'Order these posts according to the number of comments they got: least comments first',
   ORDPostLikesNumber: 'Order these posts according to the number of likes: fewest likes on top',
   ORDPostTime: 'Order theses posts after the time you posted them: oldest on top',
-  MCWhoLikedYourPost: 'Who liked this post you made?',
+  MCWhoReactedToYourPost: 'Who reacted (Like, Wow, Haha, Love, Sad, Angry, Thankful) to this post you made?',
+  MCWhoReactedToYourPostWithLIKE: 'Who liked this post you made?',
+  MCWhoReactedToYourPostWithWOW: 'Who reacted with Wow to this post you made?',
+  MCWhoReactedToYourPostWithHAHA: 'Who reacted with Haha to this post you made?',
+  MCWhoReactedToYourPostWithLOVE: 'Who reacted with Love to this post you made?',
+  MCWhoReactedToYourPostWithSAD: 'Who reacted with Sad to this post you made?',
```
Comment (Rabyss) [inline meteor/imports/client/helpers/getQuestionTitleByType.ts]: Who was sad about your post?

File: meteor/imports/client/helpers/getQuestionTitleByType.ts [hunk 5]
```diff
@@ -10,7 +10,14 @@ const en = {
   ORDPostCommentsNumber: 'Order these posts according to the number of comments they got: least comments first',
   ORDPostLikesNumber: 'Order these posts according to the number of likes: fewest likes on top',
   ORDPostTime: 'Order theses posts after the time you posted them: oldest on top',
-  MCWhoLikedYourPost: 'Who liked this post you made?',
+  MCWhoReactedToYourPost: 'Who reacted (Like, Wow, Haha, Love, Sad, Angry, Thankful) to this post you made?',
+  MCWhoReactedToYourPostWithLIKE: 'Who liked this post you made?',
+  MCWhoReactedToYourPostWithWOW: 'Who reacted with Wow to this post you made?',
+  MCWhoReactedToYourPostWithHAHA: 'Who reacted with Haha to this post you made?',
```
Comment (Rabyss) [inline meteor/imports/client/helpers/getQuestionTitleByType.ts]: Who laughed at your post?

File: meteor/imports/client/helpers/getQuestionTitleByType.ts [hunk 6]
```diff
@@ -10,7 +10,14 @@ const en = {
   ORDPostCommentsNumber: 'Order these posts according to the number of comments they got: least comments first',
   ORDPostLikesNumber: 'Order these posts according to the number of likes: fewest likes on top',
   ORDPostTime: 'Order theses posts after the time you posted them: oldest on top',
-  MCWhoLikedYourPost: 'Who liked this post you made?',
+  MCWhoReactedToYourPost: 'Who reacted (Like, Wow, Haha, Love, Sad, Angry, Thankful) to this post you made?',
+  MCWhoReactedToYourPostWithLIKE: 'Who liked this post you made?',
+  MCWhoReactedToYourPostWithWOW: 'Who reacted with Wow to this post you made?',
```
Comment (Rabyss) [inline meteor/imports/client/helpers/getQuestionTitleByType.ts]: Who was impressed by your post?

File: meteor/imports/client/helpers/getQuestionTitleByType.ts [hunk 7]
```diff
@@ -10,7 +10,14 @@ const en = {
...
   ORDPostTime: 'Order theses posts after the time you posted them: oldest on top',
-  MCWhoLikedYourPost: 'Who liked this post you made?',
+  MCWhoReactedToYourPost: 'Who reacted (Like, Wow, Haha, Love, Sad, Angry, Thankful) to this post you made?',
+  MCWhoReactedToYourPostWithLIKE: 'Who liked this post you made?',
+  MCWhoReactedToYourPostWithWOW: 'Who was impressed by this post you made?',
+  MCWhoReactedToYourPostWithHAHA: 'Who laughed at this post you made?',
+  MCWhoReactedToYourPostWithLOVE: 'Who loved to this post you made?',
+  MCWhoReactedToYourPostWithSAD: 'Who was sad about this post you made?',
+  MCWhoReactedToYourPostWithANGRY: 'Who was angry at this post you made?',
+  MCWhoReactedToYourPostWithTHANKFUL: 'Who was thankful for this post you made?',
```
Comment (Rabyss) [inline meteor/imports/client/helpers/getQuestionTitleByType.ts]: This reaction was actually removed from  Facebook (it was a Mother's day thing). Please remove this.

File: meteor/imports/client/helpers/getQuestionTitleByType.ts [hunk 8]
```diff
@@ -10,7 +10,14 @@ const en = {
   ORDPostCommentsNumber: 'Order these posts according to the number of comments they got: least comments first',
   ORDPostLikesNumber: 'Order these posts according to the number of likes: fewest likes on top',
   ORDPostTime: 'Order theses posts after the time you posted them: oldest on top',
-  MCWhoLikedYourPost: 'Who liked this post you made?',
+  MCWhoReactedToYourPost: 'Who reacted (Like, Wow, Haha, Love, Sad, Angry, Thankful) to this post you made?',
+  MCWhoReactedToYourPostWithLIKE: 'Who liked this post you made?',
+  MCWhoReactedToYourPostWithWOW: 'Who was impressed by this post you made?',
+  MCWhoReactedToYourPostWithHAHA: 'Who laughed at this post you made?',
+  MCWhoReactedToYourPostWithLOVE: 'Who loved to this post you made?',
```
Comment (Rabyss) [inline meteor/imports/client/helpers/getQuestionTitleByType.ts]: Remove "to"

File: meteor/imports/common/models/questions/common/QuestionType.ts [hunk 9]
```diff
@@ -1,6 +1,13 @@
...
+export type QuestionType = "MCWhoMadeThisCommentOnYourPost" |
     "MCWhichPageDidYouLike" |
+    "MCWhoReactedToYourPost" |
+    "MCWhoReactedToYourPostWithLIKE" |
+    "MCWhoReactedToYourPostWithWOW" |
+    "MCWhoReactedToYourPostWithHAHA" |
+    "MCWhoReactedToYourPostWithLOVE" |
+    "MCWhoReactedToYourPostWithSAD" |
+    "MCWhoReactedToYourPostWithANGRY" |
+    "MCWhoReactedToYourPostWithTHANKFUL" |
```
Comment (Rabyss) [inline meteor/imports/common/models/questions/common/QuestionType.ts]: Thankful

File: meteor/imports/common/models/questions/common/QuestionType.ts [hunk 10]
```diff
@@ -13,9 +20,16 @@ export type QuestionType = "MCWhoLikedYourPost" |
...
   MCWhoMadeThisCommentOnYourPost: 'MCWhoMadeThisCommentOnYourPost' as QuestionType,
   MCWhichPageDidYouLike: 'MCWhichPageDidYouLike' as QuestionType,
+  MCWhoReactedToYourPost: 'MCWhoReactedToYourPost' as QuestionType,
+  MCWhoReactedToYourPostWithLIKE: 'MCWhoReactedToYourPostWithLIKE' as QuestionType,
+  MCWhoReactedToYourPostWithWOW: 'MCWhoReactedToYourPostWithWOW' as QuestionType,
+  MCWhoReactedToYourPostWithHAHA: 'MCWhoReactedToYourPostWithHAHA' as QuestionType,
+  MCWhoReactedToYourPostWithLOVE: 'MCWhoReactedToYourPostWithLOVE' as QuestionType,
+  MCWhoReactedToYourPostWithSAD: 'MCWhoReactedToYourPostWithSAD' as QuestionType,
+  MCWhoReactedToYourPostWithANGRY: 'MCWhoReactedToYourPostWithANGRY' as QuestionType,
+  MCWhoReactedToYourPostWithTHANKFUL: 'MCWhoReactedToYourPostWithTHANKFUL' as QuestionType,
```
Comment (Rabyss) [inline meteor/imports/common/models/questions/common/QuestionType.ts]: Thankful

**Comment by romac** [comment]
@vincenzobaz @Rabyss Can I merge this?

**Comment by Rabyss** [comment]
No, I'll handle it soon.

**Comment by romac** [comment]
Ok :)

**Pull request merged**